programs/benchzstd.c

```diff
@@ -139,52 +139,61 @@ static const size_t maxMemory = (sizeof(size_t) == 4)
         return r;                                      \
     }
 
-/* replacement for snprintf(), which is not supported by C89
- * sprintf() would be the supported one, but it's labelled unsafe,
- * so some modern static analyzer will flag it as such, making it unusable.
- * formatString_u() replaces snprintf() for the specific case where there are only %u arguments */
+static size_t uintSize(unsigned value)
+{
+    size_t size = 1;
+    while (value >= 10) {
+        size++;
+        value /= 10;
+    }
+    return size;
+}
```

Comment:
This solution has an admirable simplicity, but there are reasonably compact, if non-obvious, fast solutions available:

https://lemire.me/blog/2021/06/03/computing-the-number-of-digits-of-an-integer-even-faster/

Reply:
Nice reference !
This could be useful in a hot conversion to ascii loop